chirp/drivers/uv5r.py

```diff
@@ -663,20 +687,21 @@ def _do_upload(radio):
             addr = 0x1808 + (i - 0x1EC0)
             _send_block(radio, i, mmap[addr:addr + 0x10])
 
-    if not image_matched_radio:
-        msg = ("Upload finished, but the 'Other Settings' "
-               "could not be sent because the firmware "
-               "version of the image (%s) does not match "
-               "that of the radio (%s).")
-        raise errors.RadioError(msg % (image_version, radio_version))
-
     if radio._all_range_flag:
         radio._all_range_flag = False
         LOG.warning('Sending all ranges to radio has completed')
         raise errors.RadioError(
             "This is NOT an error.\n"
             "The upload has finished successfully.\n"
             "Please restart CHIRP.")
+        return
+
+    if radio._aux_block and not imagev_matched_radiov:
+        msg = ("This is NOT an error. The upload finished successfully.\n"
+               "The 'Other Settings' and 'Service Settings' were skipped "
+               "because the firmware version of the image (%s) does not "
+               "match that of the radio (%s).")
+        raise errors.RadioError(msg % (image_version, radio_version))
```

Comment:
We need to figure out a better way to do this. I need to do some thinking on it, so don't let me forget.

Reply:
Yeah. The problem here is that Baofeng pretty much stopped separating the various models by using different magics. So it is entirely possible to upload an image from a dual-band radio into a tri-band radio hosing the band limits. It is a similar situation between radios with 2 TX power levels and 3 TX power levels.

Reply:
What I mean is a better way to raise a message to the user after cloning that isn't an error. So that you don't have to say "ERROR: This is not an error" :)

Reply:
Yeah. I've never liked this. People see the error symbol and never read what it says and always thing something bad happened. You are the one that suggested that I add the "This is not an error" part.

Reply:
Yeah, I remember. I'll do some thinking.